job_definitions/smoke_tests.yml

```diff
@@ -39,15 +39,15 @@
                   text(name: 'URL', value: "<${BUILD_URL}smoke_test_report|${BUILD_DISPLAY_NAME}>")
                 ]
         }
-
-        node {
-          git url: 'git@github.com:alphagov/digitalmarketplace-functional-tests.git', branch: 'master', credentialsId: 'github_com_and_enterprise'
-          stage('Run smoke tests') {
-            try {
+        try {
+          node {
+            stage('Prepare') {
+              git url: 'git@github.com:alphagov/digitalmarketplace-functional-tests.git', branch: 'master', credentialsId: 'github_com_and_enterprise'
```

Comment:
Are we going to include any `wipe-workspace` flags or repository cleaning (I see in a couple of other jobs we do combos of `git clean -fdx` and `git reset --hard origin/master`)?

Reply:
It feels a bit like treating the symptom rather than the cause. We shouldn't _have_ to include it. Lets chat in the 10 o'clock.

Reply:
I double checked and I think we only do the git clean etc for the `generate_upload_and_notify_counterpart_signature-pages` job (which is a bit of a unique case, it's the only regular script of ours not run in a Docker image). Let's try without it.

Reply:
Is this good for an approve?

Reply:
Yep looks good.